draft-ietf-scitt-architecture.md

```diff
@@ -331,12 +331,15 @@ Before an Issuer is able to produce Signed Statements, it must first create its
 A DID can be *resolved* into a *key manifest* (a list of public keys indexed by a *key identifier*) using many different DID methods.
 
 Issuers MAY choose the DID method they prefer, but with no guarantee that all Transparency Services will be able to register their Signed Statements.
-To facilitate interoperability, all Transparency Service implementations SHOULD support the `did:web` method {{DID-WEB}}.
+To facilitate interoperability, all Transparency Service implementations MUST support the `did:web` method {{DID-WEB}}.
```

Comment:
One method is "mandatory" to support, other DID Methods are optional, a transparency service MUST support at least 1 did method.

Reply:
For interoperability there are a few choices: either we pick a DID method that is mandatory to implement or we have the TS support all methods listed in the document and clients can select one.

Which approach do you prefer?

Reply:
@hannestschofenig we discussed wanting a dedicated issue to discuss, and keeping this PR on track.

Reply:
@ad-l  to file a separate issue

Reply:
https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/98